swath_projector/utilities.py

```diff
@@ -43,10 +43,14 @@ def get_variable_values(
         return make_array_two_dimensional(variable[:])
     elif 'time' in input_file.variables and 'time' in variable.dimensions:
         # Assumption: Array = (1, y, x)
-        return variable[0][:].filled(fill_value=fill_value)
+        return transpose_if_xdim_less_than_ydim(
+            variable[0][:].filled(fill_value=fill_value)
+        )
     else:
         # Assumption: Array = (y, x)
-        return variable[:].filled(fill_value=fill_value)
+        return transpose_if_xdim_less_than_ydim(
```

Comment:
I was talking about the line that says 
```
        # Assumption: Array = (y, x)
```

Reply:
Perhaps clarify the comment to "First assume Array = (along-track, across-track), but transpose if along-track.size < across-track.size.  Similarly for line 45.

Reply:
To keep it more concise, I thought it would be best to modify the comments to:
line 45: # Assumption: Array = (time, along-track, across-track)
line 50: # Assumption: Array = (along-track, across-track)

Fixed in commit [78dd9df](https://github.com/nasa/harmony-swath-projector/pull/19/commits/78dd9df6bb3ca5e408928a1c5cf3ac51a21004d7)